pcsx2/IopCommon.h

```diff
@@ -28,7 +28,7 @@
 #include "IopCounters.h"
 #include "IopSio2.h"
 #include "IopGte.h"
-static const s64 PSXCLK = 36864000;	/* 36.864 Mhz */
+static s64 PSXCLK = 36864000;	/* 36.864 Mhz */
```

Comment:
`PSXCLK ` should not be `static`. There'll be multiple copies of the variable since it's in a header file, and as a result of that any value modifications will not be global (it'll only affect some uses of the variable), which I don't think is intended in this case.

Reply:
That's how I found it, I can amend that though.